extensions/sql-database-projects/src/models/project.ts

```diff
@@ -242,23 +251,39 @@ export class Project {
 	}
 
 	private addDatabaseReferenceToProjFile(entry: DatabaseReferenceProjectEntry): void {
-		const referenceNode = this.projFileXmlDoc.createElement(constants.ArtifactReference);
-		referenceNode.setAttribute(constants.Condition, constants.NetCoreCondition);
-		referenceNode.setAttribute(constants.Include, entry.isSystemDatabase ? entry.fsUri.fsPath.substring(1) : entry.fsUri.fsPath); // need to remove the leading slash for system database path for build to work on Windows
+		let referenceNode = this.projFileXmlDoc.createElement(constants.ArtifactReference);
+
+		// if it's a system database reference, we'll add an additional node with the SSDT location of the dacpac
+		if ((<SystemDatabaseReferenceProjectEntry>entry).ssdtUri) {
+			referenceNode.setAttribute(constants.Condition, constants.NetCoreCondition);
+		}
+
+		referenceNode.setAttribute(constants.Include, (<SystemDatabaseReferenceProjectEntry>entry).ssdtUri ? entry.fsUri.fsPath.substring(1) : entry.fsUri.fsPath); // need to remove the leading slash for system database path for build to work on Windows
```

Comment:
not sure what this check means... if it has ssdt uri then we use substring other not?